tests/integration/site-details/same-activity-description/same-activity-description.test.js

```diff
@@ -0,0 +1,234 @@
+import { JSDOM } from 'jsdom'
+import { getByLabelText, getByRole, getByText } from '@testing-library/dom'
+import { createServer } from '~/src/server/index.js'
+import { statusCodes } from '~/src/server/common/constants/status-codes.js'
+import {
+  getExemptionCache,
+  updateExemptionMultipleSiteDetails
+} from '~/src/server/common/helpers/session-cache/utils.js'
+import { validateErrors } from '~/tests/integration/shared/expect-utils.js'
+
+jest.mock('~/src/server/common/helpers/session-cache/utils.js')
+
+describe('Same activity description page', () => {
+  let server
```

Comment:
The `setupTestServer` and `mockExemption` helpers (example in the activity dates integration test) will remove a lot of this setup code